practice_independent_challenges/week2/potluck/spec/potluck_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rspec'
+require './lib/dish'
+require './lib/potluck'
+
+describe Potluck do 
+    describe '#initialize' do
+        it 'has a date' do
+            potluck = Potluck.new("7-13-18")
+            expect(potluck.date).to eq "7-13-18"
+        end 
+
+        it 'has no dishes by default' do
+            potluck = Potluck.new("7-13-18")
+            expect(potluck.dishes).to eq []
+        end 
+        
+    end
+
+    describe '#add_dish' do
+        it 'returns a list of dishes' do
+            potluck = Potluck.new("7-13-18")
+            couscous_salad = Dish.new("Couscous Salad", :appetizer)
+            cocktail_meatballs = Dish.new("Cocktail Meatballs", :entre)
+            potluck.add_dish(couscous_salad)
+            potluck.add_dish(cocktail_meatballs)
+            expect(potluck.dishes).to eq ["Couscous Salad", "Cocktail Meatballs"]
```

Comment:
Looking at the interaction pattern, we actually want the entire dish object to be returned and not just the names.
```
pry(main)> potluck.dishes
#=> [#<Dish:0x00007fccc4249940...>, #<Dish:0x00007fccc499ceb8...>]
```